test/safe/branch.test.ts

```diff
@@ -38,9 +38,15 @@ describe("Branch Webhook", () => {
 		await sqsQueues.branch.purgeQueue();
 	});
 
-	describe("Create Branch", () => {
+	describe.each([true, false])("Create Branch - New GH Client feature flag is '%s'", (useNewGithubClient) => {
```

Comment:
Made sure both Octokit and Github client variation get tested, catch it before deployment!!